### ISSUE TITLE
Missing strip program

### ISSUE THREAD

**Issue by ghost** [haskell/cabal#2241]
Using cabal.git 6c395bb I see the following message:

```
'setup-Configure-Cabal-1.18.1.3-x86_64-linux-ghc-7.8.3' (missing the 'strip'
program)
```

```
$ which strip
/usr/bin/strip
```

Is this safe to ignore or a bug?

**Comment by hvr** [comment]
Fwiw, I'm seeing this too with GHC 7.10.1RC2 and cabal-install-1.22

**Comment by 23Skidoo** [comment]
@tibbe, I think this should go into the 1.22 branch.

**Comment by tibbe** [comment]
@23Skidoo I saw that you merged it into 1.22 yourself.
